tests/rustdoc-gui/sidebar-mobile.goml

```diff
@@ -57,6 +57,7 @@ scroll-to: ".block.keyword li:nth-child(1)"
 compare-elements-position-near: (".block.keyword li:nth-child(1)", ".mobile-topbar", {"y": 544})
 
 // Now checking the background color of the sidebar.
+reload:
```

Comment:
Why did you add this?

Reply:
it doesn't work without it (or a press-key: "Escape"), since we can no longer open the settings menu while the sidebar is open.

Reply:
Then instead use:

```
// Close the sidebar menu.
press-key: "Escape"
```